docs/proposals/configmap.md

```diff
@@ -293,12 +296,12 @@ spec:
         - name: config-map-volume
           mountPath: /mnt/config-map
   volumes:
-  - name: config-map-volume
-    configMap:
-      files:
-        - path: "etc/redis.conf"
-          configMapName: redis-volume-config
-          key: redis.conf
+    - name: config-map-volume
+      configMap:
+        configMapName: redis-volume-config
```

Comment:
@thockin ugh, you're right, but that's a bug.  `SecretName` should not be `omitempty`, since it is required.

Anyway, I'm fine with using LocalObjectReference now, but I thought we wanted to make this like existing volumes.  For example, we used `Items` for the list of projections into files because that's what downward API volume uses.  If we use LocalObjectReference, we should use `files` instead of `items`, too.  What do you say?